src/languageservice/services/yamlCompletion.ts

```diff
@@ -324,34 +328,8 @@ export class YamlCompletion {
         }
       }
 
-      let originalNode = node;
+      const originalNode = node;
       if (node) {
-        // when the value is null but the cursor is between prop name and null value (cursor is not at the end of the line)
-        if (isMap(node) && node.items.length && isPair(node.items[0])) {
```

Comment:
the mistake was to take only 0. index. 
null node could be anywhere inside the collection. So there would have to be some complicated implementation to find the closest null node.

Same problem inside array fix